community/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,119 @@
+ - **Be collaborative**: The success and spirit of SourceCred depends on the participation of many people. Effective and friendly collaboration enables everyone to better achieve their goals.  
+ - **Be aware**: We are careful and respectful in our communication, and we take responsibility for our own speech. Be aware of how what you say or do might impact others. Be kind to others. Do not insult or put down other members of the community.
+
+<br>
+
+## Unacceptable Behavior
+
+We, the members of the SourceCred community, are committed to making participation in this community a harassment-free experience.
+
+We will not accept harassment or other exclusionary behaviors, such as:
+
+ * The use of sexualized language or imagery  
+ * Excessive profanity (please avoid curse words; people differ greatly in their sensitivity to swearing)  
+ * Posting sexually explicit or violent material
+ * Violent or intimidating threats or language directed against another person or group
+ * Inappropriate physical contact and/or unwelcome sexual attention or sexual comments
+ * Sexist, racist, or otherwise discriminatory jokes and language
+ * Trolling or insulting and derogatory comments
+ * Unwarranted exclusionary comments, such as based on experience level, disability or appearance
+ * Continuing behavior that makes someone uncomfortable after being asked to stop.
+ * Disclosure of personal information of another person without explicit permission
+ * Other unethical or unprofessional conduct
+ * Sustained disruption of talks, events, or communications, such as heckling of a speaker
+ * Advocating for, or encouraging, any of the above behaviors
+
+<br>
+
+## Reporting Guidelines
+
+For possibly unintentional breaches of the code of conduct, you may want to respond to the person and point out this code of conduct (either in public or in private, whatever is most appropriate). 
+If you believe someone is violating the code of conduct and have decided to submit a report, please report this in a timely manner. Code of conduct violations reduce the value of the community for everyone. The SourceCred community takes reports of misconduct very seriously
+and are committed to preserving and maintaining the welcoming nature of our community.
+All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. 
+
+### Confidentiality
+
+All reports will be kept confidential, with the exception of cases where the Benevolent Dictator determines the report should be shared with law enforcement. In those cases, the report will be shared with the proper legal authorities.
+
+In some cases the Benevolent Dictator may determine that a public statement will need to be made. If that’s the case, the identities of all involved parties and reporters will remain confidential unless those individuals instruct us otherwise.
```

Comment:
> will remain confidential unless those individuals instruct us otherwise.

I think the BD should make a judgement call first whether it's helpful to include identities.
So rather than "instruct" I would use "gave permission" along the lines of:
> will remain confidential, unless the reporter gave permission to use their identity for this purpose.

Reply:
Fixed locally.